zebra/zebra_mpls_vty.c

```diff
@@ -67,11 +67,6 @@ static int zebra_mpls_transit_lsp(struct vty *vty, int add_cmd,
 		return CMD_WARNING_CONFIG_FAILED;
 	}
 
-	if (gate_str == NULL) {
```

Comment:
You're quite right that we need to keep supporting all of the valid 'no' forms - but won't this change lead to a failure below where `inet_ntop()` is called?

Reply:
Sorry for it. I have modified it.  (refer to commit  a44e310631)